src/ConnectedMode/Binding/ImportBeforeFileGenerator.cs

```diff
@@ -19,32 +19,39 @@
  */
 
 using System;
+using System.ComponentModel.Composition;
 using System.IO;
 using System.IO.Abstractions;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Integration;
 
 namespace SonarLint.VisualStudio.ConnectedMode.Binding
 {
-    internal class ImportBeforeFileGenerator
+    public interface IImportBeforeFileGenerator
```

Comment:
Does this need to be `public`? Could the interface and class both be `internal`?

Reply:
The interface should have a doc comment explaining what it does